test/UnitTest/persistance/test_persistance.py

```diff
@@ -0,0 +1,128 @@
+import os,sys
+sys.path.append('/root/GO/src/persistance/fogflow/test/UnitTest/Persistance')
+from datetime import datetime
+import copy
+import json
+import requests
+import time
+import pytest
+import data
+import sys
+
+# change it by broker ip and port
+designerIp="http://180.179.214.208:8080"
```

Comment:
[minor]better than hard coding it, it should be read from any config file or fetches from the running broker task.